website/docs/r/sentinel_data_connector_microsoft_threat_intelligence.html.markdown

```diff
@@ -0,0 +1,90 @@
+---
+subcategory: "Sentinel"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_sentinel_data_connector_microsoft_threat_intelligence"
+description: |-
+  Manages a Microsoft Threat Intelligence Data Connector.
+---
+
+# azurerm_sentinel_data_connector_microsoft_threat_intelligence
+
+Manages a Microsoft Threat Intelligence Data Connector.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "east us"
+}
+
+resource "azurerm_log_analytics_workspace" "test" {
+  name                = "example-workspace"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_sentinel_log_analytics_workspace_onboarding" "example" {
+  resource_group_name = azurerm_resource_group.example.name
+  workspace_name      = azurerm_log_analytics_workspace.example.name
+}
+
+resource "azurerm_sentinel_data_connector_microsoft_threat_intelligence" "example" {
+  name                                         = "example-dc-msti"
+  log_analytics_workspace_id                   = azurerm_log_analytics_workspace.example.id
+  bing_safety_phishing_url_lookback_date       = "1970-01-01T00:00:00Z"
+  microsoft_emerging_threat_feed_lookback_date = "1970-01-01T00:00:00Z"
+
+  depends_on = [
+    azurerm_sentinel_log_analytics_workspace_onboarding.test
+  ]
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Microsoft Threat Intelligence Data Connector. Changing this forces a new Microsoft Threat Intelligence Data Connector to be created.
+
+* `log_analytics_workspace_id` - (Required) The ID of the Log Analytics Workspace. Changing this forces a new Data Connector to be created.
+
+* `bing_safety_phishing_url_lookback_date` - (Optional) The lookback date for the Bing Safety Phishing Url in RFC3339. Changing this forces a new Data Connector to be created.
+
+* `microsoft_emerging_threat_feed_lookback_date` - (Optional) The lookback date for the Microsoft Emerging Threat Feed in RFC3339. Changing this forces a new Data Connector to be created.
+
+-> **NOTE:** At least one of `bing_safety_phishing_url_lookback_date` and `microsoft_emerging_threat_feed_lookback_date` must be specified.
+
+---
+
+* `tenant_id` - (Optional) The ID of the tenant that this Microsoft Threat Intelligence Data Connector connects to. Changing this forces a new Microsoft Threat Intelligence Data Connector to be created.
+
+-> **NOTE** Currently, only the same tenant as the running account is allowed. Cross-tenant scenario is not supported yet.
```

Comment:
If Cross-tenant is not supported, maybe this property should not be exposed in current version?

Reply:
I think we can keep it to keep consistency with other data connectors?